Clarify age verification for Voice Chat users (#1552)
Roblox/creator-docs: content/en-us/chat/voice-chat.md

---

<Alert severity="success">
Voice Chat is currently available to all 13+ age-verified users in a specific set of countries. Review the [Voice Chat FAQ](https://en.help.roblox.com/hc/en-us/articles/4405807645972-Voice-Chat-FAQs) to learn more. Users **not** in these countries should use [ID verification](https://en.help.roblox.com/hc/en-us/articles/4407282410644) to enable chat with voice. Once verified, eligible 13+ users can opt‑in to use this feature by visiting their account **Settings** page or from within a voice enabled game, allowing them to chat with voice in any Roblox game that supports it. Games with voice often see an uplift in engagement, DAU, and spending.
</Alert>

> 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

1. <Chip label="OPTIONAL" size="small" variant="outlined" /> For greater communication among players within your game, toggle on **Enable&nbsp;Camera** to allow eligible players to animate their avatar with their movement.
1. [Publish](../production/publishing/publish-games-and-places.md) the place to save the changes.

> By enabling this setting, eligible 13+ users can opt-in to Voice Chat within any voice chat supported game. Enabling Voice Chat is required to use the speech-to-text API.

Reviewer: ```suggestion
By enabling this setting, eligible 13+ users can opt-in to Voice Chat within any game that supports the feature. Enabling Voice Chat is required to use the speech-to-text API.
```
